src/toxcore/dht/dht_friend.rs

```diff
@@ -57,54 +57,46 @@ impl DhtFriend {
     }
 
     /// get Socket Address list of a friend, a friend can have multi IP address bacause of NAT
-    pub fn get_addrs_of_clients(&self, is_ipv6_mode: bool) -> Vec<SocketAddr> {
+    pub fn get_addrs_of_clients(&self) -> Vec<SocketAddr> {
+        let mut sock_v4 = Vec::new();
```

Comment:
It seems we don't need to split IPv4 and IPv6 addresses here. `c-toxcore` does this because of flaky implementation.